CONFIG/config.cpp

```diff
@@ -203,10 +205,13 @@ bool CConfigApp::ValidateSettings()
 			m_3d_video_ram = TRUE;
 			is_modified = TRUE;
 		}
+		/*	Forces full screen whenever m_flip_surfaces is enabled, and it almost always is.
+		/*	This isn't relevant to isle-portable, so we can just comment it out.
 		if (!m_full_screen) {
 			m_full_screen = TRUE;
 			is_modified = TRUE;
 		}
+		*/
```

Comment:
It makes sens because you can't flip the page if not in full screen, but it's all mostly automated by the compositor at this point so you can just remove it.

```suggestion
```